doc/src/user_guide.rst

```diff
@@ -93,6 +96,39 @@ To manage installation of Python dependencies we strongly recommend
 using `pip <https://pypi.python.org/pypi/pip>`_ and
 `virtualenv <https://pypi.python.org/pypi/virtualenv>`_.
 
+Quick-start Mac OS X
+^^^^^^^^^^^^^^^^^^^^
+
+To quickly install PyFR on a Mac we recommend using the package manager
+`homebrew <https://brew.sh/>`_.
+Open the terminal and install the dependencies with the following commands::
+
+    brew install python3 cmake open-mpi openblas metis
+    brew cask install paraview
```

Comment:
Any particular reason we are suggesting people get paraview from brew rather than downloading the app?

Reply:
no particular reason other than being the quickest way of getting a working version.

We can mention in the text that alternatively one can download a given version of paraview from the website.